src/agri_ai/core/agent.py

```
@@ -221,10 +298,76 @@ def add_user_context(self, user_id: str, context: Dict[str, Any]):
+        start_time = time.time()
+        error_occurred = False
+        
+        try:
+            agent = await self.get_agent(user_id)
+            response = await agent.process_message(message, user_id)
+            return response
+            
+        except Exception as e:
+            error_occurred = True
+            logger.error(f"Error processing message for user {user_id}: {e}")
+            return f"申し訳ございません。処理中にエラーが発生しました。もう一度お試しください。"
```

Ruff (0.11.9)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)